rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -223,7 +262,12 @@ def _decode_repay(
 
     def _decode_lending_pool_events(self, context: DecoderContext) -> DecodingOutput:
         """Decodes AAVE V2 Lending Pool events"""
-        if context.tx_log.topics[0] not in (ENABLE_COLLATERAL, DISABLE_COLLATERAL, DEPOSIT, WITHDRAW, BORROW, REPAY):  # noqa: E501
+        if context.tx_log.topics[0] not in (LIQUIDATION_CALL, ENABLE_COLLATERAL, DISABLE_COLLATERAL, DEPOSIT, WITHDRAW, BORROW, REPAY):  # noqa: E501
+            return DEFAULT_DECODING_OUTPUT
+
+        if context.tx_log.topics[0] == LIQUIDATION_CALL:
```

Comment:
nitpick: to stay in line with the other branches of this condition, it could have been just an `if` without `return DEFAULT_DECODING_OUTPUT`.

The only branch here where output is returned explicitly is `return DecodingOutput(event=event)`, because there is an event to be returned. But in other branches, there is no special return and  `DEFAULT_DECODING_OUTPUT` is just returned at the very end.

Reply:
The return is there to avoid executing extra logic that is not needed. In this case avoiding a potential query to the globaldb for the token and then executing the checks in the if statements.